config/wpe_config.toml

```diff
@@ -65,14 +65,17 @@ ias_url = "https://api.trustedservices.intel.com/sgx/dev"
 ias_api_key = "112233445566778899aabbccddeeff00"
 
 # TEE enclave library to use.
-enclave_library = "libwpe-enclave.signed.so"
+enclave_library = "libtcf-enclave.signed.so"
 enclave_library_path = "tc/sgx/trusted_worker_manager/enclave/build/lib/"
 
+# The type of enclave viz - KME/WPE/SWE
+enclave_type = "WPE"
+
 # -------------------------------------------------------------
 # TCS Worker configuration details
 # -------------------------------------------------------------
 
 [KMEListener]
 # URL of KMEListener where a Work order processor can send requests meant for
 # the Key Management Enclave(KME)  
-bind = "http://kme-listener:1948" 
+bind = "http://avalon-kme:1948"
```

Comment:
You need to add instructions to README.md on how to configure this for standalone mode.
That is, add instructions to change this line to http://localhost:1948

Reply:
This config param is for WPE to connect to KME Listener. bind make sense for KME config file. For WPE config , kme_listener_url is more appropriate